syncer/syncer.go

```diff
@@ -2334,88 +2351,98 @@ func (s *Syncer) handleQueryEvent(ev *replication.QueryEvent, ec eventContext, o
 		return nil
 	}
 
-	usedSchema := string(ev.Schema)
-	parser2, err := event.GetParserForStatusVars(ev.StatusVars)
+	qec := &queryEventContext{
+		eventContext:    &ec,
+		ddlSchema:       string(ev.Schema),
+		originSQL:       utils.TrimCtrlChars(originSQL), // Will TrimCtrlChars affect code?
```

Comment:
i think `TrimCtrlChars` doesn't  affect originSQL, it just trims invalid char from the sql string, but i am wondering why we need this func to handle sql in binlog? need some suggestions from other reviewer

Reply:
but if we trim sql here, we don't need to trim it again in `parseDDLSQL`

Reply:
pingcap/tidb-enterprise-tools#258